<launch>

    <include file="$(find sensors)/launch/usv_simulation.launch" />

    <node pkg="usvspeedcontroller" type="usvspeedcontroller_node" name="usvspeedcontroller_node" />

    <node pkg="sensors" type="trial_1.py" name="trial_1" />

</launch>
